airflow/contrib/sensors/azure_cosmos_sensor.py

```diff
@@ -1,67 +1,64 @@
-# -*- coding: utf-8 -*-

```

Comment:
Sorry for that, but this file contains invalid end-of-line characters.
Diff by charachers is available:
![Screenshot 2019-08-07 at 03 39 45](https://user-images.githubusercontent.com/12058428/62588240-57a4d880-b8c5-11e9-809d-501b4132224d.png)